src/java/org/apache/cassandra/cql3/CQL3Type.java

```diff
@@ -598,6 +672,11 @@ public static Raw tuple(List<CQL3Type.Raw> ts)
             return new RawTuple(ts);
         }
 
+        public static Raw vector(CQL3Type.Raw t, int dimention)
```

Comment:
dimention? Either this is a typo for dimension, or it's something I don't know about.